hack/images/dcos/README.md

```diff
@@ -4,29 +4,28 @@ This package contains scripts to build the DCOS Kubernetes docker container imag
 
 ![Kubernetes-Mesos on DCOS](images/dcos_k8sm.png)
 
-## DNS Support
+## Usage
 
-Kubernetes supports an add-on called kube-dns.
-kube-dns is intended to resolve Kubernetes service names, such as `frontend.default.kubernetes.local`, that map to internal Kubernetes service portal IP addresses.
-These service portal IP addresses are managed by the Kubernetes cluster and traffic is routed to them via special iptables DNAT rules that are maintained by the k8s proxy.
-Service portal IP addresses are not intended to be referenced by anything other than other k8s pods/services (read: no other mesos client or task should reference them).
+```shell
+$ make clean
+$ make build
+$ make push
+```
 
-Deploying the kube-dns addon is optional (defaults to on) and it can be disabled by setting the 'enable-dns' package option (see the walkthrough example below).
-There is no funny interaction between kube-dns and mesos-dns;
-mesos-dns servers may be used as fallbacks for name resolution since their IP addresses are present in the slave's `/etc/resolv.conf`.
+If you source directory is not clean, this will be reflected in the Docker tag with "dirty" and "unclean".
 
-## Usage
+Optionally the version might be specified:
+
+```shell
+$ GIT_REF=v1.0.5-v0.6.1 make build
+$ GIT_REF=v1.0.5-v0.6.1 make push
 ```
-## build and push the docker image
-:; make release FROM=/k8sm-build-output-dir
 
-## promote the latest -dev build to a version-tagged release
-## (don't forget to update the universe config.json/version and package.json/docker-image)
-:; make promote
+Moreover, there are the following variables can be customized via the command line:
 
-## HOWEVER, to generate a fresh release image and push it up (skip the -dev tag cycle):
-## (this is probably an exceptional case, images should usually be promoted from -dev)
-:; make release FROM=~/bin RELEASE=1
+```make
+GIT_REPO ?= mesosphere/kubernetes
```

Comment:
We default to our repo because we use our tags. Will fix it to be able to use a custom url.